.gitmodules

```diff
@@ -1,28 +1,30 @@
-[submodule "sorc/ufs_model.fd"]
-  path = sorc/ufs_model.fd
-  url = https://github.com/ufs-community/ufs-weather-model
-  ignore = dirty
 [submodule "sorc/wxflow"]
   path = sorc/wxflow
   url = https://github.com/NOAA-EMC/wxflow
-[submodule "sorc/gfs_utils.fd"]
-  path = sorc/gfs_utils.fd
-  url = https://github.com/NOAA-EMC/gfs-utils
-[submodule "sorc/ufs_utils.fd"]
-  path = sorc/ufs_utils.fd
-  url = https://github.com/ufs-community/UFS_UTILS.git
-[submodule "sorc/verif-global.fd"]
-  path = sorc/verif-global.fd
-  url = https://github.com/NOAA-EMC/EMC_verif-global.git
 [submodule "sorc/gsi_enkf.fd"]
   path = sorc/gsi_enkf.fd
   url = https://github.com/NOAA-EMC/GSI.git
```

Comment:
The GSI hash in this PR does not point to a commit in the authoritative repo. 
![Screenshot 2024-03-22 at 2 51 54 PM](https://github.com/NOAA-EMC/global-workflow/assets/11394126/98e433fc-9627-45a8-b3f3-5557857b14fd)

Reply:
Updated hash to [dfb958fa9](https://github.com/NOAA-EMC/GSI/commit/dfb958fa9372c10c808d20e64f2955652017ee32).

Reply:
@HenryWinterbottom-NOAA Please reset the order of the submodules in this file so it shows just the addition of the upp submodule. Thanks!